src/officehours_ui/templates/base.html

```diff
@@ -83,20 +83,28 @@
       </div>
     </nav>
   </header>
-  
-  {% for page in flatpages %}
-    {% if page.title == "User Comm Banner" %}
-      {{page.content|safe}}
-    {% endif %}
-  {% endfor %}
 
-	<div id="main-content" class="container" role="main">
+  <div id="main-content" class="container" role="main">
     {% block content %}{% endblock %}
     {% csrf_token %}
 	</div>
 
   <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js" integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3" crossorigin="anonymous"></script>
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.min.js" integrity="sha384-cuYeSxntonz0PPNlHhBs68uyIAVpIIOZZ5JqeqvYYIcEL727kskC66kF92t6Xl2V" crossorigin="anonymous"></script>
   {% block scripts %}{% endblock %}
+
+  <footer class="sticky-footer">
+    {% for page in flatpages %}
+      <div class="alert alert-info" role="alert">
```

Comment:
I've noticed that the alert class' styling comes with margin-bottom:1rem by default, which leaves behind a small space below the footer. I'd suggest you overwrite it so we don't have that dead space on screen